srcpkgs/nextcloud-client/template

```diff
@@ -34,6 +35,8 @@ fi
 subpackages="nextcloud-client-kwallet nextcloud-client-devel"
 subpackages+=" $(vopt_if dolphin 'nextcloud-client-dolphin')"
 
+make_check="ci-skip"
```

Comment:
move this to the first block, remove " and add a comment why this is necessary if you know

Reply:
thanks for the review. please find an updated version.

Reply:
`make_check=ci-skip` already means the tests are failing only in CI, so you don't need to leave a comment

Reply:
ok, cleaned